prisma/seed.ts

```diff
@@ -1,12 +1,13 @@
 import { PrismaClient } from '@prisma/client';
+import { encodePassword } from '../src/utils/bcrypt';
```

Comment:
**praise**: Importing encodePassword is a good step towards securing user passwords in the seed data.